portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/CustomEventsEventAnalysis.testcase

```diff
@@ -280,6 +280,119 @@ definition {
 		}
 	}
 
+	@description = "Bug ID: LRAC-11659 | Automation ID: LRAC-11726 | Test Summary: Check Default Events Counted as Default when event sent by console"
+	@priority = "5"
+	test CheckDefaultEventsCountedAsDefaultWhenEventSentByConsole {
+		var layoutName = "Custom 1";
+		var globalAttributesList = "canonicalUrl,pageTitle,referrer,url";
+
+		var disableAttributesList = "${globalAttributesList}";
+		var attributesList = "category,like";
+
+		task ("Navigation to default events tab") {
+			ACUtils.launchAC();
+
+			ACNavigation.goToSettings();
+
+			ACSettings.goToDefinitions();
+
+			ACSettings.goToEvents();
+
+			ACNavigation.openItem(itemName = "Default Events");
+		}
+
+		task ("Access the assetViewed") {
+			ACNavigation.openItem(itemName = "assetViewed");
+		}
+
+		task ("Copy the sample JS and replace an existing attribute with a new one") {
+			ACCustomEvents.copySampleJavascript(
+				attributesList = "${attributesList}",
+				disableAttributesList = "${disableAttributesList}");
+		}
+
+		task ("Close sessions") {
+			ACUtils.closeAllSessions();
+
+			ACUtils.launchDXP();
```

Comment:
> Hey guys! I'm having trouble pasting the javascript code on console browser. The value of sampleJavascript passing in for executJSInBrowserConsole macro gave me an error. I'm not sure if the format for sampleJSScript is correct. Here is the gist of error log. https://gist.github.com/calvinctvo/5c4bf685df8c74094c43b850086edc15

I think it's because you need to navigate to the site that is connected with AC. You're only launching DXP but still need to navigate to the pages with connected AC. So would probably need to add 
`ACUtils.navigateToSitePage(
				pageName = "AC Page",
				siteName = "Site Name");`